hack/jenkins/minikube_cross_build_and_upload.sh

```diff
@@ -23,18 +23,27 @@
 
 set -e
 
+MINIKUBE_DIR=${GOPATH}/src/k8s.io/minikube
+FILES_CHANGED=$(git diff ${ghprbActualCommit} --name-only $(git merge-base origin/master ${ghprbActualCommit}))
+
 gsutil cp gs://minikube-builds/logs/index.html gs://minikube-builds/logs/${ghprbPullId}/index.html
 
 # If there are ISO changes, build and upload the ISO
 # then set the default to the newly built ISO for testing
-if out="$(git diff ${ghprbActualCommit} --name-only $(git merge-base origin/master ${ghprbActualCommit}) | grep deploy/iso/minikube)" &> /dev/null; then
+if out="$(${FILES_CHANGED} | grep deploy/iso/minikube)" &> /dev/null; then
```

Comment:
Makes sense, I guess it would be nice to put it somewhere else because then it would be easier to run locally, but I don't think it's a huge deal.